apps/mull-api/src/database/seeders/database.seeder.ts

```diff
@@ -0,0 +1,28 @@
+import { Connection } from 'typeorm';
```

Comment:
Please also run `nx run-many --target=lint --all` and fix the linting issues you introduced where possible: 
![image](https://user-images.githubusercontent.com/23544999/104862249-ef06bb80-5900-11eb-902e-221a38397ec0.png)

If some of the rules seems ridiculous lemme know, we could disable them so that they don't annoy us in the future

Reply:
I'll try to rename some of the variables but the 'createFriends' was used before it was defined seems weird, is it because I have to have the function above it?

Reply:
Maybe, I'm not sure